addons/osfstorage/listeners.py

```diff
@@ -24,7 +24,7 @@ def checkin_files_task(node_id, user_id):
         Guid = apps.get_model('osf.Guid')
         OSFUser = apps.get_model('osf.OSFUser')
 
-        node = Guid.load_referent(node_id)
+        node, _ = Guid.load_referent(node_id)
```

Comment:
Heads-up, this piece has been merge in https://github.com/CenterForOpenScience/osf.io/pull/10852. You may run into conflicts for this PR.